### DISCUSSION TITLE
ui.select() dropdown width unexpected behaviour

### DISCUSSION THREAD

**Discussion by inessilva4-lis** [zauberzeug/nicegui#3342] [Q&A]
### Description

Hello! I am facing an issue with the dropdown width of ui.select(). The width of the dropdown depends on the size of the options rather than the input field. When the list of options is long and the strings have varying lengths, the width of the dropdown keeps changing. Is there a way to define a maximum width for the dropdown so that the options wrap to two or more lines when they exceed the set width?

**Comment by falkoschindler** [comment]
Hi @inessilva4-lis,

> the width of the dropdown keeps changing

I'm not sure if I understand correctly. Can you give an example?

> Is there a way to define a maximum width for the dropdown so that the options wrap to two or more lines when they exceed the set width?

[Quasar's QSelect](https://quasar.dev/vue-components/select) has a "popup-content-class" prop for styling the popup:

```py
ui.select([
    'Very long option that should wrap',
    'foo',
    'bar',
    'baz',
]).props('popup-content-class="max-w-[200px]"')
```
<img width="215" alt="Screenshot 2024-07-11 at 17 06 19" src="https://github.com/zauberzeug/nicegui/assets/5767091/fdae005a-e981-4680-8ec4-09f85c3b050b">
